src/main/kotlin/org/koitharu/kotatsu/parsers/site/nepnep/NepnepParser.kt

```diff
@@ -242,14 +244,14 @@ internal class Manga4Life(context: MangaLoaderContext) : MangaParser(context, Ma
 		)
 		val pageTotal = curChapter.getString("Page")!!.toInt()
 		val host = "https://" +
-			script
-				.substringAfter("vm.CurPathName = \"", "")
-				.substringBefore('"')
-				.also {
-					if (it.isEmpty()) {
-						throw Exception("Manga4Life is overloaded and blocking Tachiyomi right now. Wait for unblock.")
+				script
+					.substringAfter("vm.CurPathName = \"", "")
+					.substringBefore('"')
+					.also {
+						if (it.isEmpty()) {
+							throw Exception("Manga4Life is overloaded and blocking kotatsu right now. Wait for unblock.")
```

Comment:
Avoid throwing so generic exceptions. It might be a `IllegalStateException` or `ParseException`

Reply:
And the application name must be capitalized)